Test/Misc.lean

```diff
@@ -22,8 +22,9 @@ data QpfList₃ α where
 data PairOf α β
   | mk : α → β → β → PairOf α β
 
-data QpfTest α β where
-  | A : α → α → β → QpfTest α β → QpfTree β → QpfCoTree (QpfTree (QpfTest α β)) → QpfTest α β
+-- FIXME: this broke after Upgrading to Lean/Mathlib 4.5.0 and the associated changes
+-- data QpfTest α β where
+--   | A : α → α → β → QpfTest α β → QpfTree β → QpfCoTree (QpfTree (QpfTest α β)) → QpfTest α β
```

Comment:
I filed an issue (#2) with my thoughts on this issue, and how to solve it.
I'm fine with merging this and implementing the fix later, so don't worry about it

Reply:
Sounds good, thanks! :)